rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -1564,6 +1552,22 @@ JobIdData submitFromUrl(@HeaderParam("sessionid") String sessionId, @HeaderParam
             throws JobCreationRestException, NotConnectedRestException, PermissionRestException,
             SubmissionClosedRestException, IOException;
 
+    /**
+     * Submits a list of workflows to the scheduler from a list of workflow URLs
+     *
+     * @param sessionId
+     *            a valid session id
+     * @param jsonBody
+     *            a list of workflows. Each element must contain an url, a set of variables and generic information
+     * @return the list of objects containing, for each workflow a <code>jobid</code> if submission was successful, or the error description if an error occurred.
+     */
+    @POST
+    @Path("{path:jobs}/body/multi")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    List<JobIdDataAndError> multipleSubmitFromUrls(@HeaderParam("sessionid") String sessionId,
+            List<WorkflowUrlData> jsonBody) throws NotConnectedRestException;
```

Comment:
@fviale would you please give an example of the JSON body of the request. Thanks in advance

Reply:
Input:
```
[
  {
    variables: {variables values set at submission ...
      string =>string
    }
    genericInformation: {generic information values set at submission ...
      string =>string
    }
    workflowUrl:stringUrl of the workflow that will be fetched ...
  }
]
```
OUTPUT:
```
[
  {
    stackTrace:string
    id:number
    error:boolean
    errorMessage:string
    readableName:string
    fetchError:boolean
  }
]
```